etcdctl/ctlv3/command/auth_command.go

```diff
@@ -40,6 +41,14 @@ func newAuthEnableCommand() *cobra.Command {
 	}
 }
 
+func newAuthenticateCommand() *cobra.Command {
```

Comment:
I also thought it isn't useful. But I found this issue https://github.com/coreos/etcd/issues/4603
If grpc proxy will be used in the future, this kind of command maybe useful? I'm still not sure.